search/langfind/langfind.md

```diff
@@ -12,7 +12,7 @@ tag=reddit json Body | langfind -e Body | count by lang | sort by count desc | t
 
 ### Supported Options
 
-* `-e <arg>`: The “-e” option operates on an enumerated value instead of on the entire record. For example, a pipeline that performed language analysis on HTTP payloads would be `tag=pcap ipv4.DstPort==80 tcp.Payload | langfind -e Payload`
+* `-e <arg>`: The “-e” option operates on an enumerated value instead of on the entire record. For example, a pipeline that performed language analysis on HTTP payloads would be `tag=pcap ipv4.DstPort==80 tcp.Payload | langfind -e Payload`.
```

Comment:
Note that when this is displayed, there will not be single ticks. It's too easy for a user to copy-paste `langfind -e Payload.` if they're not paying attention.

Reply:
@floren PTAL